content/uk/docs/reference/glossary/storage-class.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Storage Class
```

Comment:
```suggestion
title: StorageClass
```
These two words are not found in Ukrainian.

Reply:
Here, I added the translated term into AKA 🤷‍♂️ and left the original title.